.tx/config

```diff
@@ -0,0 +1,109 @@
+[main]
+host = https://www.transifex.com
+
+[qgis-model-baker-docs.index-md]
```

Comment:
@marioba is there a possibility to generalize this? it seems this file is very likely to be forgotten when new chapters are added to the doc

Reply:
Yes we can. I will add the command:
```
tx config mapping-bulk -p qgismodelbaker-docs --source-language en --type MD -f '.md' --source-file-dir docs --expression "docs/{filepath}/{filename}.<lang>{extension}" --execute
```
to the workflow